sources/packages/web/src/components/aest/AssessApplicationOfferingChangeRequestModal.vue

```diff
@@ -0,0 +1,119 @@
+<template>
+  <v-form ref="assessApplicationOfferingChangeRequestForm">
+    <modal-dialog-base
+      :showDialog="showDialog"
+      :title="getTitle"
+      :max-width="780"
+    >
+      <template #content
+        ><error-summary
+          :errors="assessApplicationOfferingChangeRequestForm.errors"
+        />
+        <p class="my-4">
+          Outline the reasoning for
+          <span
+            v-if="
+              showParameter ===
+              ApplicationOfferingChangeRequestStatus.DeclinedBySABC
+            "
+            >declining</span
+          ><span v-else>approving</span>
+          this request. Please add the application number.
+        </p>
+        <p class="label-bold mb-1">Notes</p>
+        <v-textarea
+          variant="outlined"
+          hide-details="auto"
+          v-model="note"
+          :rules="[checkNotesLengthRule]"
+          required
+        />
+        <p class="pt-1 brand-gray-text">
+          Notes will be visible to StudentAid staff and institutions. This will
+          not be shown to students.
+        </p>
+      </template>
+      <template #footer>
+        <footer-buttons
+          :primaryLabel="getPrimaryLabel"
+          @secondaryClick="cancel"
+          @primaryClick="assessChange"
+        />
+      </template>
+    </modal-dialog-base>
+  </v-form>
+</template>
+<script lang="ts">
+import ModalDialogBase from "@/components/generic/ModalDialogBase.vue";
+import { ref, defineComponent, computed } from "vue";
+import { useModalDialog, useRules } from "@/composables";
+import { ApplicationOfferingChangeAssessmentAPIInDTO } from "@/services/http/dto";
+import { ApplicationOfferingChangeRequestStatus, VForm } from "@/types";
+
+export default defineComponent({
+  components: {
+    ModalDialogBase,
+  },
+  setup() {
+    const note = ref("");
+    const { showDialog, showModal, resolvePromise, showParameter } =
+      useModalDialog<ApplicationOfferingChangeAssessmentAPIInDTO | boolean>();
+    const assessApplicationOfferingChangeRequestModal = ref(
```

Comment:
Do we need this variable created as a `ref` or do we need this variable at all?
Can you please clarify why is it returned in the setup method?